packages/esm-service-queues-app/src/queue-table/default-queue-table.component.tsx

```diff
@@ -79,15 +97,24 @@ function DefaultQueueTable() {
           <ExtensionSlot
             name="patient-search-button-slot"
             state={{
+              isOpen: isPatientSearchOpen,
```

Comment:
I think we need to make similar changes to other places where the `patient-search-button-slot` is used:

esm-appointments-app/src/metrics/metrics-header.component.tsx
esm-service-queues-app/src/views/queue-tables-for-all-statuses.component.tsx

Reply:
The `isOpen` is an optional prop, it is only used where needed.